src/Drips.sol

```diff
@@ -65,6 +69,10 @@ library Drips {
     /// @notice Maximum number of drips receivers of a single user.
     /// Limits cost of changes in drips configuration.
     uint8 internal constant MAX_DRIPS_RECEIVERS = 100;
+    /// @notice The additional decimals for all amtPerSec values.
+    uint8 internal constant AMT_PER_SEC_EXTRA_DECIMALS = 18;
+    /// @notice The multiplier for all amtPerSec values. It's `10 ** AMT_PER_SEC_EXTRA_DECIMALS`.
+    uint256 internal constant AMT_PER_SEC_MULTIPLIER = 1_000_000_000_000_000_000;
```

Comment:
That was the initial implementation (`10**AMT_PER_SEC_EXTRA_DECIMALS` to be exact), but it breaks completely on the inline assembly. It ONLY tolerated constants which are literals, because any expression may or may not get precalculated during compilation, it's up to solc.

An obvious workaround is to copy the constant to a local variable right before the assembly block, but that turns out to explode gas usage, for the worst case of 100 receivers by 1.3 M gas or by almost 16%.

Reply:
Ah interesting. Okay, then lets keep it.